ci/templates/ci/identities_detail.html

```diff
@@ -173,6 +194,17 @@ <h2 class="dashhead-title">Identity Management</h2>
           <td>{{ subscription.completed }}</td>
           <td>{{ subscription.created_at|get_date|date:"D d M Y H:i" }}</td>
           <td>{{ subscription.updated_at|get_date|date:"D d M Y H:i" }}</td>
+          <td>
+          {% if subscription.active == True %}
```

Comment:
Only one comment from me, please have a look at the html indentation. I also think it's best to ask someone else to also do a review so that my code is also reviewed. @KaitCrawford or @rudigiesler if you have a chance please have a look at this PR, thanks.